tests/timeseries/test_periodic_feature_extractor.py

```diff
@@ -41,7 +41,11 @@ async def init_feature_extractor(
     window, sender = init_moving_window(timedelta(seconds=len(data)))
     async with window:
         await push_logical_meter_data(sender, data)
-        yield PeriodicFeatureExtractor(moving_window=window, period=period)
+        try:
+            yield PeriodicFeatureExtractor(moving_window=window, period=period)
+        except Exception as e:  # pylint: disable=broad-except
+            print(f"<init_feature_extractor> Unhandled exception caught: {e}")
+            raise
```

Comment:
Same. In this case since (without this `except`) there would be no *cleanup* code, we don't really need to add the `try/except`.